src/Symfony/Component/Lock/StoreInterface.php

```diff
@@ -51,18 +43,4 @@ public function waitAndSave(Key $key);
      * @throws LockConflictedException
      */
     public function putOffExpiration(Key $key, $ttl);
```

Comment:
belongs to `PersistStoreInterface`

Reply:
yes but for the sake of bc we do not change anything in here AFAIR